mkinitcpio

```diff
@@ -709,6 +709,8 @@ trap -- ERR
 
 # this is simply a nice-to-have -- it doesn't matter if it fails.
 ldconfig -r "$BUILDROOT" &>/dev/null
+# remove /var/cache/ldconfig/aux-cache for reproducability
```

Comment:
Hey! There is a typo in this comment. ("reproducability" -> "reproducibility")

Reply:
Thanks fixed.